modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -427,7 +437,11 @@ export default withRouter(
 							messageBoardSections.items &&
 							messageBoardSections.items.length,
 					}))
-					.then(setSection);
+					.then((section) => {
+						setSection(section);
+						setSectionQuery(getSectionsQuery);
```

Comment:
Why state for this and not just a variable?

Reply:
Just because I doubted if it was a bad practice to have a variable mutating inside the component. 

I understand that if it is not used in any hook it is better to create a variable with let and just store the value there to use it later, correct? 

I will send a follow up to change it when you confirm it